python/paddle/fluid/tests/unittests/test_pool2d_op.py

```diff
@@ -363,6 +363,10 @@ def init_adaptive(self):
         self.adaptive = False
 
 
+class TestPool2D_Op(TestPool2D_Op_Mixin, OpTest):
```

Comment:
why this one is empty?

Reply:
The multiple inheritance gives the final structure. It's a combo of mixin and OpTest classes. That way it is run by unittest as proper test